### ISSUE TITLE
postActions could be resolved after tools registration

### ISSUE THREAD

**Issue by smiklosovic** [arquillian/arquillian-spacelift-gradle#53]
Right now, in default installation, there is this (1) and after it, there is tool registration.

However, I can imagine that the installation could register tools which you want to use in post installation closure but as it is done now - they are not existing yet.

The workaround is to make custom installation which calls postActions.resolve() in registerTools() method after all tools are registered hence, to my understanding, invocation of postActions will have access to just registered tools.

(1) https://github.com/arquillian/arquillian-spacelift-gradle/blob/888196fadbbac6d6bfefadd7f62aea447adabd32/src/main/groovy/org/arquillian/spacelift/gradle/DefaultInstallation.groovy#L211

**Comment by smiklosovic** [comment]
@kpiwko

**Comment by kpiwko** [comment]
Yes, this one is problematic. However, it is also the other way around - there might be steps needed to do prior registration itself

Unclear whether we want to have 2 closures. Likely some renaming and having just one with clear behavior will do
